lib/kube/proxy/roundtrip.go

```diff
@@ -112,82 +111,7 @@ func (s *SpdyRoundTripper) Dial(req *http.Request) (net.Conn, error) {
 // dial dials the host specified by req, using TLS if appropriate, optionally
 // using a proxy server if one is configured via environment variables.
```

Comment:
I think this godoc is not accurate anymore now that this method always dials without proxy it seems.